config/connection.js

```diff
@@ -9,8 +9,8 @@ const connection = mysql.createConnection({
     user: "root",
 
     //adding process.env (accessing the environment variable password) so I don't have to show my password in code ("PASSWORD= ___ node server.js")
-    password: process.env.PASSWORD,
```

Comment:
do not store your personal password in version control. Keep using - Process.env.PASSWORD 

To use this, type `PASSWORD=yourpassword node server.js` instead of just `node server.js`